content/documentation/recipes_multisig_create_account.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Creating Multisig Accounts
+description: Recipes | Multisig | Creating Multisig Accounts | Iron Fish Documentation
+---
+
+Iron Fish supports multisig accounts using FROST, a threshold signature scheme.
+
+### Creating participant identities
+
+Each signer or participant in a multisig account must have a unique identity. The participant identity is a public key derived from a participant's secret key and allows other participants to encrypt data before sending it.
+
+The example below shows how to generate a random signer identity using the Iron Fish SDK:
+
+```js
+import { multisig } from '@ironfish/rust-nodejs'
+
+const secret = multisig.ParticipantSecret.random()
+const identity = secret.toIdentity()
+```
+
+You can also use the [`wallet/multisig/createParticipant`](/developers/documentation/rpc/wallet/multisig/create_participant) RPC to create a participant identity and store the participant secret key in the wallet.
+
+### Generating multisig accounts
+
+Once all participants have created and shared their identities you can create a multisig account for the list of participant identities using Trusted Dealer Key Generation. When creating an account the trusted dealer must also specify the minumum number of signers required to sign a transaction.
+
+In the example below we use the Iron Fish SDK to create 'key packages' for each of a list of participant identities.
+
+```js
+import { multisig } from '@ironfish/rust-nodejs'
```

Comment:
I tried setting up a simple javascript project. I don't think `multisig` is exported from the project. But the following works: 
```
import { generateAndSplitKey } from '@ironfish/rust-nodejs';
```

Reply:
I was using the wrong ironfish version, my b